pkgs/development/compilers/chicken/5/eggDerivation.nix

```diff
@@ -32,15 +38,14 @@ in
     for f in $out/bin/*
     do
       wrapProgram $f \
-        --prefix CHICKEN_REPOSITORY_PATH : "$out/lib/chicken/${toString chicken.binaryVersion}" \
-        --suffix CHICKEN_INCLUDE_PATH : "$out/share" \
+        --prefix CHICKEN_REPOSITORY_PATH : "$out/lib/chicken/${toString chicken.binaryVersion}:$CHICKEN_REPOSITORY_PATH" \
```

Comment:
I don't know anything about chicken why this variable was added ?

Reply:
I am just reverting what I did a few PRs back. I dont know why this broke and I have to investigate that, but for now this repairs some programs that come with some eggs.